### PR TITLE
feat: Priority component

### PR DESCRIPTION
**Pull request by tusharkandpal**
To check:

- Priority component

### CODE REVIEW

File: src/components/note/Note.jsx [hunk 1]
```diff
@@ -41,6 +41,19 @@ export function Note({ setShowAddNote, note, id }) {
       </div>
       <div>
         <div className="chips-list">
+          <span
+            className="chip"
+            style={{
+              backgroundColor:
+                priority === "Low"
```
Comment (ankit9015) [inline src/components/note/Note.jsx:46]: I like how you chained conditional operator

**Review by ankit9015 — APPROVED** [review]
Code is well structured and good to merge

File: src/components/note/Note.jsx [hunk 2]
```diff
@@ -41,6 +41,19 @@ export function Note({ setShowAddNote, note, id }) {
...
+                  ? "lightgreen"
+                  : priority === "Medium"
+                  ? "yellow"
+                  : "#f78080",
+            }}
```
Comment (paarthbajaj) [inline src/components/note/Note.jsx:51]: Good use of conditional operator

File: src/components/priority/Priority.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/* || PRIORITY */
```
Comment (atharvabhanage02) [inline src/components/priority/Priority.css:1]: Can remove empty spaces

File: src/components/text-editor/TextEditor.css [hunk 4]
```diff
@@ -27,6 +27,8 @@
   justify-content: center;
 }
 .editor-icons {
+  display: flex;
```
Comment (paarthbajaj) [inline src/components/text-editor/TextEditor.css:30]: You could have used .flex-row from your CL

**Review by atharvabhanage02 — APPROVED** [review]
UI looks clean and code is well structured
Preview looks fine

**Review by paarthbajaj — APPROVED** [review]
Good, simple and clean code. Liked it. You can merge this PR. Well done!

**Pull request merged**